### ISSUE TITLE
"Data URI" is a deprecated term that can still be found all around MDN (should be “data URL”

### ISSUE THREAD

**Issue by pygy** [mdn/content#15358]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL

### What specific section or headline is this issue about?

links to the data URL page

### What information was incorrect, unhelpful, or incomplete?

"Data URI" should be changed to "Data URL" across MDN

both 

- https://github.com/mdn/content/search?q=%22data+URI%22&type=code
- https://github.com/mdn/content/search?q=%22data+URIs%22&type=code

return a lot of results.

Even [the data URLs page](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs) still has "URI" in its URL, a rename/redirection would be in order.

### What did you expect to see?

"data URL"

### Do you have any supporting links, references, or citations?

https://url.spec.whatwg.org/ whose logic has been applied to the content of https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs, but not the rest of MDN

**Comment by pygy** [comment]
Would this be the way to do it?

```shell
$ git mv ./files/en-us/web/http/basics_of_http/data_uris ./files/en-us/web/http/basics_of_http/data_urls
```
```shell
$ yarn content add-redirect /en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs /en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs
```

Then do a global find/replace.

**Comment by cw118** [comment]
It looks like at least part of this has been fixed with #16031 and #16074 (cf. #16012, a similar/related issue), though I didn't replace all occurrences of the term.
